src/pages/_id/index.vue

```diff
@@ -245,6 +248,19 @@ export default {
     userAvatar() {
       return this.userInfo && this.userInfo.avatar;
     },
+    sortedOwnedNFTClassId() {
+      const sorted = [...this.ownedNFTClassId];
+      sorted.sort((a, b) => {
```

Comment:
Merge into 1 line? The first line's array is not sorted and is named so now
```suggestion
      const sorted = [...this.ownedNFTClassId].sort((a, b) => {
```